File: Front-end/health-mind-frontend/vite/src/shared/types/dtos/Recepcionista.dto.ts
import { StatusCargoEnum } from "shared/enums/statusCargo.enum"
import { StatusRoleEnum } from "shared/enums/statusRole.enum"

export interface RecepcionistaDTO {
    id?: string,
    nome: string,
    email: string,
    cpfCnpj: string
    senha: string,
    statusCargo: StatusCargoEnum,
    statusRole: StatusRoleEnum
}

File: Back-end/HealthMindBackend/HealthMindBackend.Application/DTOs/RecepcionistaDTO.cs
using HealthMindBackend.Domain.Enums;
using HealthMindBackend.Domain.ValueObjects;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HealthMindBackend.Application.DTOs
{
    public class RecepcionistaDTO
    {
        public String? Id { get; set; }
        public String Nome { get; set; }
        public String Email { get; set; }
        public String Senha { get; set; }
        public String CpfCnpj { get; set; }
        public StatusCargoEnum StatusCargo { get; set; }
        public StatusRoleEnum StatusRole { get; set; }
        public String? UsuarioId { get; set; }
        public Boolean RegenerarCredenciais { get; set; }
    }
}
